### ISSUE TITLE
Remove local preloaded repositories from "sbt.boot.properties" file.

### ISSUE THREAD

**Issue by gslowikowski** [sbtrun-maven-plugin/sbtrun-maven-plugin#12]
As part of issue #11 two repositories ware added to `sbt.boot.properties`:
```
  local-preloaded-ivy: file:///${sbt.preloaded-${sbt.global.base-${user.home}/.sbt}/preloaded/}, [organization]/[module]/[revision]/[type]s/[artifact](-[classifier]).[ext]
  local-preloaded: file:///${sbt.preloaded-${sbt.global.base-${user.home}/.sbt}/preloaded/}
```

On Windows system with spaces in `${user.home}` path (e.g. `C:/Documents and Settings/username`) when one of them is being accessed by `SBT`, build fails with exception:
```
      [sbt] java.net.URISyntaxException: Illegal character in path at index 18: file:/C:/Documents and Settings/gs/.sbt/preloaded/
      [sbt] 	at java.net.URI$Parser.fail(URI.java:2848)
      [sbt] 	at java.net.URI$Parser.checkChars(URI.java:3021)
      [sbt] 	at java.net.URI$Parser.parseHierarchical(URI.java:3105)
      [sbt] 	at java.net.URI$Parser.parse(URI.java:3053)
      [sbt] 	at java.net.URI.<init>(URI.java:588)
      [sbt] 	at java.net.URL.toURI(URL.java:946)
      [sbt] 	at sbt.Resolver$url$.apply(Resolver.scala:253)
      [sbt] 	at sbt.Classpaths$.sbt$Classpaths$$bootRepository(Defaults.scala:1549)
      [sbt] 	at sbt.Classpaths$$anonfun$appRepositories$1.apply(Defaults.scala:1524)
      [sbt] 	at sbt.Classpaths$$anonfun$appRepositories$1.apply(Defaults.scala:1524)
      [sbt] 	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
      [sbt] 	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
      [sbt] 	at scala.collection.IndexedSeqOptimized$class.foreach(IndexedSeqOptimized.scala:33)
      [sbt] 	at scala.collection.mutable.WrappedArray.foreach(WrappedArray.scala:34)
      [sbt] 	at scala.collection.TraversableLike$class.map(TraversableLike.scala:244)
      [sbt] 	at scala.collection.AbstractTraversable.map(Traversable.scala:105)
      [sbt] 	at sbt.Classpaths$.appRepositories(Defaults.scala:1524)
      [sbt] 	at sbt.Classpaths$$anonfun$40.apply(Defaults.scala:1040)
      [sbt] 	at sbt.Classpaths$$anonfun$40.apply(Defaults.scala:1040)
      [sbt] 	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
      [sbt] 	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
      [sbt] 	at sbt.EvaluateSettings$MixedNode.evaluate0(INode.scala:177)
      [sbt] 	at sbt.EvaluateSettings$INode.evaluate(INode.scala:135)
      [sbt] 	at sbt.EvaluateSettings$$anonfun$sbt$EvaluateSettings$$submitEvaluate$1.apply$mcV$sp(INode.scala:67)
      [sbt] 	at sbt.EvaluateSettings.sbt$EvaluateSettings$$run0(INode.scala:76)
      [sbt] 	at sbt.EvaluateSettings$$anon$3.run(INode.scala:72)
      [sbt] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
      [sbt] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
      [sbt] 	at java.lang.Thread.run(Thread.java:745)
```

These definitions will be removed.

**Comment by gslowikowski** [comment]
`1.0.1-SNAPSHOT` snapshot deployed.

**Comment by gslowikowski** [comment]
**IMPORTANT**

After investigating, I found, that the problem with Windows paths containing spaces was fixed in SBT version `0.13.7` with [this commit](https://github.com/sbt/sbt/commit/a857ae275905c5d75a4abae03b901a9981f6aa2c) (issue is [here](https://github.com/sbt/sbt/issues/1570)).

The exception shown above was generated by SBT `0.13.5`.

So, when using SBT `0.13.7` or later, version `1.0.0` of this plugin will work without problems.

If, for some reason, older version of SBT is required, use version `1.0.1` of this plugin.
